tests/drivers/spi/spi_loopback/prj_stm32f072b_disco.conf

```diff
@@ -0,0 +1,10 @@
+CONFIG_BOOT_BANNER=y
+CONFIG_BUILD_TIMESTAMP=y
+CONFIG_SYS_LOG=y
+CONFIG_SPI=y
+CONFIG_SPI_LEGACY_API=n
+CONFIG_SYS_LOG_SPI_LEVEL=1
+CONFIG_SPI_STM32=y
+CONFIG_SPI_STM32_INTERRUPT=y
```

Comment:
It isn't needed. It defaults to y in boards Kconfig.defconfig when SPI is enabled.